### DISCUSSION TITLE
Confused about playing Anim Montages

### DISCUSSION THREAD

**Discussion by Alex-Arg** [Sixze/ALS-Refactored#383] [Q&A]
As the title says, I'm somewhat confused about how to execute Montages.
I'm simply trying to run a full-body montage (pickup animation), and I'm experimenting with curves and slots, but I can't get the montage to play. I saw some posts here in discussions, but I haven't had any luck.

Some doubts: Should I add a "pickup" Action? When I add an anim notify with the "rolling" action to my montage, I manage to get the animation to play, though it's a bit odd.

The only way I saw to add a new action is by modifying the cpp code; is there a better way?

This is what I have right now:
![image](https://github.com/Sixze/ALS-Refactored/assets/7539939/7558ecac-3a8b-4b4d-aa6b-7fd7a4bdfa6b)

In general, any direction towards any information on how play a simple montage is really welcome, 
thank you very much.

**Comment by ameaninglessname** [comment]
Full body montage should be played at `PostLocomotion` slot, you can refer to `AM_Als_Mantle_High`.

**Comment by Alex-Arg** [comment]
Thanks so much for the help @ameaninglessname 
I just reproduce the "AM_Als_Mantle_High" setting, with no luck 😞 

I am sure I am missing something obvious, just trying to learn the system.

This is what I have now:
![image](https://github.com/Sixze/ALS-Refactored/assets/7539939/5bc40d8b-03c2-4e1a-8d27-16fd7c918ea0)

If there's something I'm missing, I'd really appreciate the help!

**Reply by ishensarabia** [reply]
You should set the values to -1 instead of 1
